MotionMark/tests/core/resources/leaves.js

```diff
@@ -23,44 +23,48 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-window.Leaf = Utilities.createSubclass(Particle,
-    function(stage)
+class Leaf extends Particle {
+    static get sizeMinimum() { return 20; }
```

Comment:
It can be just 
```
static sizeMinimum = 20;
```

Reply:
I don't believe this correctly allows the subclass to override the base class static property.